packages/suite-data/src/browser-detection/index.ts

```diff
@@ -107,6 +107,11 @@ window.addEventListener('load', () => {
             version: 69,
             mobile: true,
         },
+        {
+            name: 'Edge',
+            version: 96,
+            mobile: true,
```

Comment:
Have you tested Edge on Android?

Reply:
Apologies for the oversight here.  Checking on my Pixel 6, it appears the WebUSB is not working with Edge.  I have changed it to false in a further commit.